pkg/apis/openebs.io/v1alpha1/cstor_restore.go

```diff
@@ -35,11 +36,14 @@ type CStorRestore struct {
 
 // CStorRestoreSpec is the spec for a CStorRestore resource
 type CStorRestoreSpec struct {
-	RestoreName   string `json:"restoreName"` // set restore name
-	VolumeName    string `json:"volumeName"`
-	RestoreSrc    string `json:"restoreSrc"`
-	MaxRetryCount int    `json:"maxretrycount"`
-	RetryCount    int    `json:"retrycount"`
+	RestoreName   string            `json:"restoreName"` // set restore name
+	VolumeName    string            `json:"volumeName"`
+	RestoreSrc    string            `json:"restoreSrc"` // it can be ip:port in case of restore from remote or volumeName in case of local restore
+	MaxRetryCount int               `json:"maxretrycount"`
+	RetryCount    int               `json:"retrycount"`
+	StorageClass  string            `json:"storageClass"`
```

Comment:
q_: For these new fields, should there be `omitifempty`, to handle the reading of older cstor restore CRs.